pepper-apis/src/main/java/org/broadinstitute/ddp/export/ExportUtil.java

```diff
@@ -0,0 +1,146 @@
+package org.broadinstitute.ddp.export;
+
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.apache.commons.lang.StringUtils;
+import org.broadinstitute.ddp.client.Auth0ManagementClient;
+import org.broadinstitute.ddp.db.TransactionWrapper;
+import org.broadinstitute.ddp.db.dao.ActivityInstanceDao;
+import org.broadinstitute.ddp.db.dao.ParticipantDao;
+import org.broadinstitute.ddp.db.dto.StudyDto;
+import org.broadinstitute.ddp.model.activity.instance.ActivityResponse;
+import org.broadinstitute.ddp.model.study.Participant;
+import org.broadinstitute.ddp.util.Auth0Util;
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.core.HandleCallback;
+
+public class ExportUtil {
+
+    public static final int DEFAULT_BATCH_SIZE = 100;
+    public static final int READER_BUFFER_SIZE_IN_BYTES = 10 * 1024;
+
+    public static <R, X extends Exception> R withAPIsTxn(HandleCallback<R, X> callback) throws X {
+        return TransactionWrapper.withTxn(TransactionWrapper.DB.APIS, callback);
+    }
+
+    public static String makeExportCSVFilename(String baseFileName, Instant timestamp) {
+        DateTimeFormatter fmt = DateTimeFormatter.ofPattern("yyyyMMdd'T'HHmmssX").withZone(ZoneOffset.UTC);
+        return String.format("%s%s.csv", fmt.format(timestamp), baseFileName);
```

Comment:
This format is different from what we have before. We're missing an underscore. And the argument order is swapped.